openbas-framework/src/main/java/io/openbas/asset/AgentService.java

```diff
@@ -0,0 +1,12 @@
+package io.openbas.asset;
+
+import io.openbas.database.repository.AgentRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class AgentService {
```

Comment:
**chore:** Should be in the api (the framework package need to be removed). under io/openbas/agent OR io/openbas/service (should be discuss with the team)